docs/canvas/canvas-create-workpad.asciidoc

```diff
@@ -0,0 +1,96 @@
+[role="xpack"]
+[[canvas-create-workpad]]
+== Create a new workpad
+
+Canvas is a creative space for presenting data. You can create a Canvas workpad 
+to combine colors, shapes, and text, to bring dynamic, pixel-perfect data displays 
+to screens large and small. A workpad is your drawing space where you can add text, 
+data elements, and visual assets. 
+
+Before you start, you will need to have your own {es} data source connected in {kib}, 
+or you can use one of {kib}'s <<add-sample-data, sample data sets>>. 
+
+[float]
+=== Starting a new project
+Open *Canvas*. Click *Templates* to create a new project from a light or 
+dark template. Each template includes placeholder text, images, and styles to help you
+get started. 
+
+You can also click *Create workpad* to start from scratch.
+
+[float]
+=== Working with elements
+Elements are individual components that can be placed on a workpad, and configured
+to suit your presentation. You can use elements to add text, logos, and images.
+An element can also be connected to a data source to display live data in your workpad. 
+
+[role="screenshot"]
+image::canvas/images/canvas-element-editor.png[]
+
+
+Adding text::  
+* Click *Add element* on the top right, and select *Markdown* to add a text box to your workpad.
+* Click on the text box in the workpad to reveal the *Markdown* menu on the right hand
+side. From this menu you can add text content, and configure text styles.
+
+Adding assets:: 
+* Click *Manage assets* to import new images.
+* Select an image from your library to add an image element to your workpad.
+
+Displaying data:: 
+* Click *Add element* to choose from available data elements, such as a coordinate plot, 
+line chart, pie chart, and more. Select an element type to add it to your workpad.
+* The new element is connected to some demo data, so you will need to change its 
+data source to connect the element to the data you want to display. 
+* Click on the element in the workpad to open the sidebar editor. From this editor, 
+you to can edit an element's *Display* settings and *Data* source. 
+* You can also open the *Expression editor* on the bottom right to bypass the sidebar
```

Comment:
Since you are introducing expressions, should there be a separate section explaining what expressions are and providing an example in this doc?